kong/plugins/jwt/handler.lua

```diff
@@ -165,6 +165,15 @@ local function do_authentication(conf)
     return false, {status = 401, message = errors}
   end
 
+  -- Verify the JWT registered claims
+  if conf.maximum_expiration ~= nil and conf.maximum_expiration > 0 then
```

Comment:
The second check of this condition seems unnecessary, since you included a safeguard to `check_maximum_expiration()` already (`if max_exp <= 0`)

Reply:
I wanted to avoid calling the function as  I know it is irrelevant.